articles/manual/index.md

```diff
@@ -0,0 +1,96 @@
+
+## How to learn Q#?
+
+### Pre-required knowledge to use Q#
+
+There is no need to be an expert on quantum physics to start writing your first quantum programs. However, there are some concepts that are essential to understand quantum computing:
+
+* **Complex numbers:** quantum mechanics makes use of complex numbers to express quantum states. Some experience with complex numbers is crucial to
+understand the basics of quantum computation.
+* **Linear algebra (vectors and matrices):** quantum states are vectors 
+  in a vector space over the complex numbers. Quantum computing makes 
+  intense use of linear operators acting on these states. Therefore 
+  some level of expertise in linear algebra is indispensable to use Q#.
+* **Dirac notation:** most quantum computing texts use Dirac notation 
+  to express the mathematical concepts. Dirac notation is just a simple 
+  notation for linear algebra widely used in quantum physics.
+* **Basic concepts of quantum mechanics:** while it is not required to
+  have a deep understanding of quantum mechanics to use Q#, it is highly
+  recommended to be familiar with some basic concepts, namely
+  *quantum state*, *qubit*, *observable*, *measurement*, *Hamiltonian*,
+  *superposition*, *entanglement*, *interference*, among others.  
+
+### Learning materials
+
+Quantum computing is an emerging field and fortunately, there is an immense amount of learning resources online. Sometimes so many options can get overwhelming, so here we gather a few of them.
+
+#### Quantum computing concepts
+
+On this list you can find resources for learning concepts related with quantum computation:
+
+* A good place to start is our guide for [Quantum computing concepts](xref:microsoft.quantum.concepts.intro), a compilation of basic concepts for quantum computing.
+* The book *Learn Quantum Computing with Python and Q#* (Sarah C. Kaiser and Christopher E. Granade) provides an excellent introduction for people who have little to no experience with quantum mechanics, but some programming background.
+* MIT OpenCourseWare has an excellent [online course](https://www.youtube.com/playlist?list=PLUl4u3cNGP61-9PEhRognw5vryrSEVLPr) imparted by Allan Adams for learning the basics of quantum mechanics.
+* In parternship with Microsft's quantum team, Brilliant has launched an interactive 
+  course called ["Quantum Computing"](https://cloudblogs.microsoft.com/quantum/2019/05/23/microsoft-brilliant-team-up-to-offer-quantum-curriculum/).
+* The book *Quantum Computation and Quantum Information* (Michael A. Nielsen, Isaac L. Chuang) being the most cited text in the field of quantum computation is regarded as the standard text on the subject. The book assumes minimal prior experience with quantum mechanics and with computer science. It is an excellent choice for those readers who want a rigorous introduction to the topic as well as for those who are looking for references for advanced concepts.
+
```

Comment:
There are a lot of overlap contents with the 5 questions - "What is Q#?" and "How do I learn quantum computing?"